personio-personnel-data-api-oa3.yaml

```diff
@@ -1267,6 +1277,186 @@ paths:
                       code: 404
                       message: 'The attendance period was not found.'
 
+  /company/attendances/projects:
+    parameters:
+      - $ref: "#/components/parameters/XPersonioPartnerID"
+      - $ref: "#/components/parameters/XPersonioAppID"
+    get:
+      security:
+        - BearerAuth: []
+      tags:
+        - Projects
+      description: Provides a list of all company projects.
+      responses:
+        "200":
+          description: List of projects returned
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                  data:
+                    type: array
+                    items:
+                      $ref: "#/components/schemas/Project"
+    post:
+      security:
+        - BearerAuth: []
+      tags:
+        - Projects
+      description: Creates a project into the company account
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - name
+              properties:
+                name:
+                  type: string
+                  example: "A project name"
+                active:
+                  type: boolean
+                  example: true
+                  default: false
+      responses:
+        "200":
+          description: Project created successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                  data:
+                    $ref: "#/components/schemas/Project"
+        "400":
+          description: Invalid data was sent
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                    example: false
+                  error:
+                    type: object
+                    properties:
+                      name:
+                        type: array
+                        items:
+                          type: string
+                          example: "Project name cannot be empty"
+
+  /company/attendances/projects/{id}:
+    delete:
```

Comment:
Now that I think about this, it looks dangerous. 

DELETE `/company/projects/{id}` makes it obvious that you are deleting a project on a company level. Meaning, I would be more cautious performing that action

DELETE `company/attendances/projects/{id}` even though it was the description, can create an illusion, that this deletion will affect only the attendances. 

Damn, the same applies to patch as well ... We need to align on those semantics more carefully at least for V2